features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/components/APIDetailsTopMenu.jsx

```diff
@@ -116,6 +116,9 @@ const APIDetailsTopMenu = (props) => {
     const {
         classes, theme, api, isAPIProduct, imageUpdate, intl,
     } = props;
+    const history = useHistory();
+    const prevLocation = history.location.pathname;
+    const lastIndex = prevLocation.split('/')[3];
```

Comment:
is it guaranteed to have the expected `lastIndex`  all the time as the 3rd element (index) ?

Reply:
Yes, Path - /apis/ff50a691-4946-4405-ae6a-2aca43df4e78/endpoints